go.mod

```
@@ -0,0 +1,5 @@
+module task.scheduler
```

Hi @Sagi-levi ,
regarding the module naming- the dot in `task.scheduler` is typically reserved for domain-like prefixes in module paths (e.g., `go.onelayer.dev/x/component`), not within the descriptive component. 
Since our module is currently independent and not designed to be fetched as a dependency, a simpler path like `task-scheduler` is valid. 


```suggestion
module task-scheduler
```

https://go.dev/doc/modules/layout
https://go.dev/ref/mod